xml/System.Security.Cryptography.Pkcs/Pkcs12Info.xml

```diff
@@ -112,10 +125,27 @@
         <Parameter Name="password" Type="System.ReadOnlySpan&lt;System.Char&gt;" />
       </Parameters>
       <Docs>
-        <param name="password">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="password">The password to use to attempt to verify integrity.</param>
+        <summary>Attempts to verify the integrity of the <see cref="P:System.Security.Cryptography.Pkcs12.Pkcs12Info.AuthenticatedSafe"/> contents with a password represented by a <see cref="T:System.ReadOnlySpan{System.Char}"/>.</summary>
+        <returns><see langword="true"/> if the password successfully verifies the integrity of the <see cref="P:System.Security.Cryptography.Pkcs12.Pkcs12Info.AuthenticatedSafe"/> contents; <see langword="false"/> if the password is not correct or the contents have been altered.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+ It is not possible to distingush the error of the password being incorrect from the error of the contents having been altered.
+
+ It is usually the case that a password-protected PFX uses the same password for the MAC algorithm providing integrity and the encryption algorithm providing confidentiality,
+ but the file format does not make that a requirement.
+ Therefore, there is no guarantee that a password which results in a `true` return from this method will succeed on a call to <xref:System.Security.Cryptography.Pkcs12.Pkcs12SafeContents.Decrypt%2A>.
+
+ In the PKCS#12 specification a distinction is made between a `null` password and an "empty" password, and that difference is reflected in the return value of this method.
+ <xref:System.ReadOnlySpan{System.Char}.Empty> represents the `null` password and `String.Empty.AsSpan()` represents the "empty" password. If one of those two values returns `true` the other will not.
+
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">The <see cref="P:System.Security.Cryptography.Pkcs12.Pkcs12Info.IntegrityMode"/> value is not <see cref="F:System.Security.Cryptography.Pkcs12.Pkcs12IntegrityMode.Password"/>.</exception>
+        <exception cref="T:System.Security.Cryptography.CryptographicException">The hash algorithm option specified by the PKCS#12 PFX contents could not be identified or is not supported by this platform.</exception>
       </Docs>
     </Member>
     <Member MemberName="VerifyMac">
```

Comment:
@mairaw or @rpetrusha : This file doesn't have a MethodGroup entry for this overloaded member (and I feel like I've seen other new API in that state).  I don't know if that means a) we're not bothering with them, b) there's an error in the ingestor which results in it not producing them c) there's an error in the validator that makes it not report them as missing d) other e) some multiple combination of a-d.

Reply:
You mean the MemberGroup element to add summary, remarks, etc. that apply to all overloads?
If so, the reflection tool doesn't add them automatically and we're not required to document them.

When we want to avoid repeating the same text, we manually add them.

If I misunderstood, please let me know.

Reply:
It seems, though, that the \<summary> element of a member group is ignored if there aren't some  \<AssemblyInfo> elements present.